aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -47,6 +48,10 @@ module aptos_framework::object {
     /// Explicitly separate the GUID space between Object and Account to prevent accidental overlap.
     const INIT_GUID_CREATION_NUM: u64 = 0x4000000000000;
 
+    #[test_only]
```

Comment:
Id remove the test only annotation as its not publicly accessible anyway. And its nice to see all these domain separators related to object creation in the same place

Reply:
Done.